tips-tricks/TroubleshootingandTips.rst

```diff
@@ -679,20 +679,22 @@ You need to load data from a structured datastore.
 Solution
 --------
 
-There are two major Nifi processors for importing data into Kylo (i.e. into Hadoop): GetTableData and ImportSqoop.
+There are two major NiFi processors for importing data into Kylo (i.e. into Hadoop): GetTableData and ImportSqoop.
 
-a. **GetTableData** leverages JDBC to pull data from the source into the flowfile within Nifi. This content will then need to be pushed to HDFS (via PutHDFS processor) to the landing area in HDFS.
+a. **GetTableData** leverages JDBC to pulls data from the source into the flowfile within NiFi. This content will then need to be pushed to HDFS (via PutHDFS processor) to the landing area in HDFS.
```

Comment:
The earlier 'pull' in lieu of 'pulls' was correct. This change should be reverted.